### DISCUSSION TITLE
Custom Markup: Experiments ...

### DISCUSSION THREAD

**Discussion by TiddlyTweeter** [wikilabs/plugins#97] [Custom Markup]
I started this discussion to begin to explore ways to use  @pmario's Custom Markup at a more advanced level. 
I am particularly interested in ...

- ... any methods to be able to DYNAMICALLY  CHANGE in text ID's from VIEW, not edit.
- ... any methods to be able to DYNAMICALLY  CHANGE in text PARAMETERS (when using the "macrocall method") from VIEW, not edit.

I will likely first post _questions_. 
Hopefully, later, _solutions_.

Best wishes, TT

**Comment by TiddlyTweeter** [comment]
### Experiment No. 1 - Dynamically Change CM ID's?

**BACKGROUND:** `\customise` rules handle DEPENDENCIES very well! 
The end user does not need to worry at all when the CM design is good.

To be able to dynamically change from VIEW the ID saves a lot of hassle switching on and off stylesheets.

So, to the **ISSUE:** I got very interested in either dynamically changing _(via transclusion???)_ markup ID's, maybe via the [TW toggle operator](https://tiddlywiki.com/#toggle%20Operator)??? OR, perhaps , in the "in text" parameter values?

I hope the question is clear!

I wonder if there is an answer :-).
TT

**Reply by pmario** [reply]
>I hope the question is clear!

Not really. ... You want to toggle behaviour in "view mode" ... IMO the macro may be responsible for this, based on a `$:/state/` or `$:/temp/` tiddler. ... But I'm not sure.

**Reply by TiddlyTweeter** [reply]
> > @TiddlyTweeter I hope the question is clear!
> 
> @pmario Not really. 

Right! What I'll do is to play a bit to see if I can form some kind of proof of concept that will show what I mean more clearly!

Best, TT
